README.md

```diff
@@ -193,24 +221,52 @@ The following instructions assume a Unix-like shell (e.g. bash).
       go env GOROOT GOPATH
 
 - Ensure `$GOPATH/bin` is on your `$PATH`.
+
+- Clone the dcrdata repository. It **must** be cloned into the following directory.
+
+      git clone https://github.com/decred/dcrdata $GOPATH/src/github.com/decred/dcrdata
+
+Once Golang is installed and the dcrdata project is cloned proceed to the build steps.
+
+### Build from source for golang >= 1.11
+
+Golang 1.11 introduced a new [dependency management feature](https://github.com/golang/go/wiki/Modules) that
+negates the use of third party tooling such as `dep`. Using `Go mod` will be the dependency managment tool of choice
+for dcrdata and other Decred projects going forward.
+
+Usage is simple and nothing is required except Golang 1.11.
+
+1. `go build`
+
+If you want to pre staging dependencies without building you can run
+
+1. `go mod download`
+
+Golang's builtin dependency management resolution will start processing
+the source code and downloading dependencies on the fly and then update the `go.mod` and `go.sum` files automatically.
+
+### Build from source for golang < 1.11
+
+Building with the `dep` dependency management tool.
+
 - Install `dep`, the dependency management tool. The current [released binary of
   `dep`](https://github.com/golang/dep/releases) is recommended, but the latest
   may be installed from git via:
 
       go get -u -v github.com/golang/dep/cmd/dep
 
-- Clone the dcrdata repository. It **must** be cloned into the following directory.
-
-      git clone https://github.com/decred/dcrdata $GOPATH/src/github.com/decred/dcrdata
-
 - Fetch dependencies, and build the `dcrdata` executable.
 
       cd $GOPATH/src/github.com/decred/dcrdata
       dep ensure -vendor-only
       # build dcrdata executable in workspace:
       go build
 
-To build with the git commit hash appended to the version, set it as follows:
+_Note_ we recommend upgrading to [Golang 1.11](#build-from-source-for-golang--111) or using the docker [container build instructions](#building-dcrdata-with-docker)
+
+### Build from commit hash
```

Comment:
The word *from* is a bit misleading as it suggests the suggestion will tell you how to choose a certain commit hash to build from.  Might as well be more specific and say it's building with the git commit hash in the version string.